docs/core/misc/slip0039.md

```diff
@@ -101,5 +102,6 @@ This process does not use passphrase.
 2. Use `slip39.recover_ems(shares)` to combine the shares and get metadata.
 3. Store the Encrypted Master Secret as `_MNEMONIC_SECRET`.
 4. Store the identifier as `_SLIP39_IDENTIFIER`.
```

Comment:
only if not extendable

Reply:
https://github.com/trezor/trezor-firmware/pull/3825/commits/5bafd9611183e9eab044ad7632077823cdb21197